back/assets/myoback_body.xml

```diff
@@ -0,0 +1,713 @@
+        <site name="MF_m5t.3_r_MF_m5t.3_r-P2" pos="-0.0328228 0.015247 0.00232083"/>
+        <site name="MF_m3.laminar_r_MF_m3.laminar_r-P1" pos="-0.0187003 0.0374421 0.01617"/>
+        <site name="MF_m5.laminar_r_MF_m5.laminar_-P2" pos="-0.0258996 0.01705 0.00564"/>
+        <site name="MF_m1t.1_l_MF_m1t.1_l-P1" pos="-0.0186347 0.0369697 -0.01759"/>
+        <site name="MF_m2s_l_MF_m2s_l-P1" pos="-0.0185029 0.0376762 -0.01581"/>
+        <site name="MF_m5s_l_MF_m5s_l-P2" pos="-0.0301638 0.01539 -0.00393"/>
+        <site name="MF_m5t.1_l_MF_m5t.1_l-P2" pos="-0.03551 0.016863 -0.00293"/>
+        <site name="MF_m5t.2_l_MF_m5t.2_l-P2" pos="-0.0345516 0.0159287 -0.00293"/>
+        <site name="MF_m5t.3_l_MF_m5t.3_l-P2" pos="-0.0328228 0.015247 -0.00277"/>
+        <site name="MF_m3.laminar_l_MF_m3.laminar_l-P1" pos="-0.0187003 0.0374421 -0.01617"/>
+        <site name="MF_m5.laminar_l_MF_m5.laminar_l-P2" pos="-0.0258996 0.01705 -0.00564"/>
+        <body name="Abdomen" pos="0.098 0.071 0">
+          <inertial pos="0 0 0" mass="0.01" diaginertia="0.001 0.001 0.001"/>
+          <joint name="Abs_t1" pos="0 0 0" axis="1 0 0" limited="false" type="slide" range="-100000 100000"/>
+          <joint name="Abs_t2" pos="0 0 0" axis="0 1 0" limited="false" type="slide" range="-100000 100000"/>
+          <joint name="Abs_r3" pos="0 0 0" axis="0 0 1" limited="false" range="-1.7 1.7"/>
+          <geom name="Abdomen_ellipsoid" size="0.01" group="2" rgba="0.19 0.83 0.78 0.2"/>
+          <site name="Abdomen_sidesite" pos="0.011 0 0" size="0.0005"/>
+          <site name="EO1_r_EO1_r-P1" pos="0.0065502 0.0627431 0.0308937"/>
+          <site name="EO2_r_EO2_r-P2" pos="0.00723321 0.0329565 0.0364551"/>
+          <site name="EO3_r_EO3_r-P2" pos="0.00677141 0.0129967 0.0416346"/>
+          <site name="EO4_r_default" pos="0.00695134 -0.00599564 0.0403302"/>
+          <site name="IO1_r_IO1_r-P1" pos="0.00306951 -0.0406576 0.0453754"/>
+          <site name="IO2_r_IO2_r-P1" pos="0.00068053 -0.0115304 0.0513889"/>
+          <site name="IO3_r_IO3_r-P1" pos="0.00338096 0.0293052 0.0418774"/>
+          <site name="EO1_l_EO1_l-P1" pos="0.0065502 0.0627431 -0.03089"/>
+          <site name="EO2_l_EO2_l-P2" pos="0.00723321 0.0329565 -0.03646"/>
+          <site name="EO3_l_EO3_l-P2" pos="0.00677141 0.0129967 -0.04163"/>
+          <site name="EO4_l_EO4_l-P2" pos="0.00695134 -0.00599564 -0.04033"/>
+          <site name="IO1_l_IO1_l-P1" pos="0.00306951 -0.0406576 -0.04538"/>
+          <site name="IO2_l_IO2_l-P1" pos="0.00068053 -0.0115304 -0.05139"/>
+          <site name="IO3_l_IO3_l-P1" pos="0.00338096 0.0293052 -0.04188"/>
+        </body>
+        <body name="lumbar4" pos="0.009 0.0321 0">
+          <inertial pos="0.0170296 0.018 0" quat="0.707107 0 0 0.707107" mass="1.799" diaginertia="0.01643 0.01123 0.0052"/>
+          <joint name="L4_L5_FE" pos="0 0 0" axis="0 0 1" limited="false" range="-100000 100000"/>
```

Comment:
this range (along with many others) seems to be too big and can produce numerical instabilities. Is it also physiological to be so big?

Reply:
Fixed the ranges here to a reasonable one, and change some muscle jumping issues:
6a6f4dd49b1905f0af702a18fb94fc7a04dfdd99